sbncode/CAFMaker/FillReco.cxx

```diff
@@ -391,6 +399,37 @@ namespace caf
     }
   }
 
+  void FillTrackScatterDCA(const art::Ptr<sbn::ScatterDCA> dca,
```

Comment:
What does DCA stand for? Depending on the answer I might propose just `scatter.foo`, or `scatter.dca.foo`.

Reply:
Now you know it is Distance of Closest Approach, do you still think this should change?

Reply:
Are there likely to be other scattering variables in future? I think maybe `scatter.closestapproach.foo` or `scatter.dca.foo` with a comment on the SRDCA class and dca variable would be good.

Reply:
This all relates to a particular method of interpolating the start/end of the track and calculating the distance of closest approach from the trajectory points to the interpolated line. There are certainly other ways to calculate the scattering metrics people will be interested in calculating in the future. I will make the names more informative.